seed/utils/api_schema.py

```diff
@@ -0,0 +1,88 @@
+# !/usr/bin/env python
+# encoding: utf-8
+from rest_framework.schemas import AutoSchema
+import coreapi
+
+
+class AutoSchemaHelper(AutoSchema):
+    def __init__(self):
+        super().__init__()
+
+    def base_field(self, name, location, required, type, description):
+        """
+        Created to avoid needing to directly access coreapi within ViewSets.
+        Ideally, the cases below will be used instead of this one.
+        """
+        return coreapi.Field(
+            name,
+            location=location,
+            required=required,
+            type=type,
+            description=description
+        )
+
+    def org_id_field(self, location='query', required=True, type='integer', description='Organization ID'):
+        return coreapi.Field(
+            'organization_id',
+            location=location,
+            required=required,
+            type=type,
+            description=description
+        )
+
+    def path_id_field(self, description):
+        return coreapi.Field(
+            "id",  # matches reference name in uri path
+            location='path',
+            required=True,
+            type='integer',
+            description=description
+        )
+
+    def body_field(self, required, description):
+        return coreapi.Field(
+            'body',
+            location='body',
+            required=required,
+            type='object',
+            description=description
+        )
+
+    def form_field(self, name, required, description):
+        return coreapi.Field(
+            name,
+            location='form',
+            required=required,
+            type='object',
+            description=description
+        )
+
+    def test_field_options(self):
```

Comment:
Leaving this here for reference:

<img width="710" alt="Screen Shot 2020-04-11 at 1 58 36 PM" src="https://user-images.githubusercontent.com/30608004/79053719-90d3dc00-7bfc-11ea-8392-2f826a27e30d.png">